kolibri/plugins/learn/assets/test/views/resumable-content-grid.spec.js

```diff
@@ -0,0 +1,73 @@
+import { shallowMount } from '@vue/test-utils';
+/* eslint-disable import/named */
+import useLearnerResources, {
+  useLearnerResourcesMock,
+} from '../../src/composables/useLearnerResources';
+/* eslint-enable import/named */
+import ResumableContentGrid from '../../src/views/LibraryPage/ResumableContentGrid';
+
+jest.mock('../../src/composables/useLearnerResources');
+
+describe('when there are nodes with progress that can be resumed', () => {
+  beforeEach(() =>
+    useLearnerResources.mockImplementation(() =>
+      useLearnerResourcesMock({
+        /*
+         * moreResumableContentNodes having length would realistically mean
+         * that there are a significant number of resumableContentNodes but we
+         * rely on useSearch to handle the details of that implementation
+         */
+        resumableContentNodes: [{ node: 1 }],
+        moreResumableContentNodes: [{ node: 2 }],
+      })
+    )
+  );
+  afterEach(() => jest.clearAllMocks());
+  it('displays resumable content nodes string', () => {
+    const wrapper = shallowMount(ResumableContentGrid, {});
+    expect(wrapper.find('[data-test="recent-content-nodes-title"').element).toBeTruthy();
+  });
+
+  it('displays grid / list toggle buttons when on medium or larger screens', () => {
+    const wrapper = shallowMount(ResumableContentGrid, {
+      computed: { windowIsMedium: () => true },
+    });
+    expect(wrapper.find('[data-test="toggle-view-buttons"]').element).toBeTruthy();
+  });
+
+  it('does not show the grid / list toggle buttons when on extra small screens', async () => {
+    const wrapper = shallowMount(ResumableContentGrid, {
+      computed: { windowIsSmall: () => true },
+    });
+    expect(wrapper.find('[data-test="toggle-view-buttons"]').element).toBeFalsy();
+  });
+
+  it('displays HybridLearningCardGrid', () => {
+    const wrapper = shallowMount(ResumableContentGrid, {});
```

Comment:
What's the purpose of this test?

Reply:
Just to test that a component is loaded - I think it was already in the LibararyPage tests that I broke up into multiple components - not particularly useful so I've removed it